public/app/plugins/panel/pmm-update/hooks/usePerformUpdate.ts

```diff
@@ -32,14 +33,19 @@ export const usePerformUpdate = (): UpdateStatus => {
       let newIsUpdated = false;
 
       try {
-        const response = await getUpdateStatus({ auth_token: authToken, log_offset: logOffset });
+        const response = await getUpdateStatus({ auth_token: authToken, log_offset: logOffset, method: updateMethod });
 
         if (!response) {
           throw Error('Invalid response received');
         }
 
         const { done, log_offset, log_lines } = response;
 
+        // Check if PMM server has stopped
+        if (log_lines.some((line) => line.includes('Stopping PMM Server in container'))) {
```

Comment:
This seems a hacky method. Is this the only way we have to know is the server stopped?

Reply:
Ok, copy that 👍